### PR TITLE
Update documentation on logout

### PR DESCRIPTION
**Pull request by JakubC-projects**
Updates the documentation regarding the Single Log Out, explaining how the implement the backchannel logout (which is our current stratagegy for SLO)

### CODE REVIEW

File: _docs/signon/openid-connect.md [hunk 1]
```diff
@@ -201,83 +201,121 @@ Payload:
...
+## Requirements
 
 HTTPS (SSL) must be installed on your application!
 
 * There are multiple free services available that provide free SSL certificates (
   e.g. [Let’s Encrypt](https://letsencrypt.org/))
 * Please contact support if you have any questions regarding SSL certificates.
 
-**WordPress applications that have the ``BCC Signon`` plugin installed should not follow this tutorial (single sign-out
+**WordPress applications that have the ``BCC Signon`` plugin installed should not follow this tutorial (single logout
```
Comment (rvanoord) [inline _docs/signon/openid-connect.md:225]: "BCC Login" is the name of the new plugin (you could add a link to the releases page on github --bcc-wp repo)

File: _docs/signon/openid-connect.md [hunk 2]
```diff
@@ -201,83 +201,121 @@ Payload:
 
 ---
 
-# Single sign-out
+# Single logout
```
Comment (rvanoord) [inline _docs/signon/openid-connect.md:204]: Could also mention that the logout url needs to be registered centrally (in auth0).. ie. need to send it to it@bcc.no.

In addition, you could link to the oidc signout specification and/or the documentation from auth0

**Review by rvanoord — CHANGES_REQUESTED** [review]

File: _docs/signon/openid-connect.md (hunk 2 above)
Comment (JakubC-projects) [reply _docs/signon/openid-connect.md:204]: It is described in the Backchannel logout explanation part https://github.com/bcc-code/bcc-code.github.io/pull/51/files#diff-2882e0a12b5e36041f56bf7d45f56a55acf247f8f6a912532d45432a8d26cab3R381

Comment (JakubC-projects) [reply _docs/signon/openid-connect.md:204]: I don't think auth0 provides a documentation as it's a beta feature, but I included an architecture diagram and now added a link to the oidc documentation

File: _docs/signon/openid-connect.md (hunk 1 above)
Comment (JakubC-projects) [reply _docs/signon/openid-connect.md]: added

**Review by rvanoord — APPROVED** [review]

**Pull request merged**